pkg/scanners/trivy/trivy.go

```diff
@@ -108,8 +99,29 @@ var (
 func main() {
 	flag.Parse()
 
+	err := logger.Configure()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error setting up logger: %s", err)
+		os.Exit(generalErr)
+	}
+
+	log.Info("config", "config", *config)
+
+	userConfig := *DefaultConfig()
+	if *config != "" {
+		var err error
+		userConfig, err = loadConfig(*config)
+		if err != nil {
+			log.Error(err, "unable to read config")
+			os.Exit(generalErr)
+		}
+	}
+
+	log.Info("userConfig", "userConfig", userConfig)
+	log.Info("userConfig", "USERCONFIG", fmt.Sprintf("%#v\n", userConfig))
```

Comment:
same here re: 1 log line, ignore if this is debug

Reply:
done